canvas_modules/common-canvas/__tests__/common-properties/operators/equalsOp-test.js

```diff
@@ -55,7 +55,7 @@ describe("validating equals operator works correctly", () => {
 		expect(equals(wrap(null), wrap(null), null, controller)).to.equal(true);
 		expect(equals(wrap({ temp: "value" }), wrap({ temp: "value2" }), null, controller)).to.equal(false);
 		// pass in a function as a way to hit the default switch case
-		expect(equals(wrap(emptyFunc), null, null, controller)).to.equal(true);
+		expect(equals(wrap(emptyFunc), wrap({ temp: "value2" }), null, controller)).to.equal(true);
```

Comment:
Extra logic was added to operation to handle nulls.  This is needed to hit the default condition.